### PR TITLE
fixes spelling in include

### PR DESCRIPTION
**Pull request by randall**

### CODE REVIEW

**Comment by randall** [comment]
[approve ci centos]

**Comment by randall** [comment]
[approve ci]

File: include/tscore/ParseRules.h [hunk 1]
```diff
@@ -33,7 +33,7 @@ typedef unsigned int CTypeResult;
 
 // Set this to 0 to disable SI
 // decimal multipliers
-#define USE_SI_MULTILIERS 1
+#define USE_SI_MULTIPLIERS 1
```
Comment (scw00) [inline include/tscore/ParseRules.h]: Please fix the ParseRules.cc(in ink_atoui64) as well .

Comment (randall) [reply include/tscore/ParseRules.h]: good catch. I probably shouldn't have split this change across multiple PRs

**Comment by randall** [comment]
[approve ci autest]

**Comment by randall** [comment]
@scw00 Thanks for the help here!

**Comment by scw00** [comment]
:-)

**Pull request merged**